build-scripts/build.sh

```diff
@@ -1,10 +1,59 @@
 #!/usr/bin/env bash
-set -e
 
+postTheme2Actions()
+{
+     if [ $? -eq 0 ]
+            then
+                node cypress/support/createReports.js
+                zip -r output/reports/screenshots.zip cypress/screenshots
+                exit 0
+            else
+                echo "The Cypress tests failed - Uploading report to S3 with no latest" >&2
+                node cypress/support/createReports.js
+                zip -r output/reports/screenshots.zip cypress/screenshots
+                $UPLOADTOS3 nolatest
```

Comment:
the way this is wrote, I think, will only work if it's the 4th argument.
https://github.com/bbc/childrens-games-jenkins/commit/9e7482bc33acfcfc6334f078b4c786a5094b4744#diff-3c062d7265d94ac4bf580860782fdfa3e8f2a9e256645c9e5b17cfc629013e9a

i'm not really a fan of having this in the s3upload script though, this is kind of bespoke. might be better separated into another script

Reply:
I also think this will fail because it will upload to s3, then at the end of the job it will try to upload the built game, but the folder will already exist so it will fail.

Reply:
Ah, we've changed that script again. @AnthonyGoodchild modified it, not sure where you could see those changes. Ant have you got a link out to it?

Reply:
This was the revised script https://github.com/bbc/childrens-games-jenkins/pull/34/files

Reply:
yeah it's just not merged yet as I was reworking how the s3 upload script is called - it's got messy with all the arguments

Reply:
(the command here will currently work as the script is modified on live)